enroll_batch_students.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+import json
+from itertools import islice
+import requests
+
+from config import EDXADAPT, DEFAULT_PROBABILITIES, DEFAULT_SKILLS
+
+path_to_file = sys.argv[1]
+
+if not path_to_file:
+    print "Usage is: python enroll_batch_student.py /path/to/file/with/student_ids"
+    sys.exit()
+elif not os.path.exists(path_to_file):
+    print "File does not exist, please try again"
+    sys.exit()
+
+headers = {'Content-type': 'application/json'}
+
+users = json.loads(
+    requests.get('http://{HOST}:{PORT}/api/v1/course/{COURSE_ID}/user'.format(**EDXADAPT), headers=headers).text
+)['users']
+
+enrolled_students = set(users['finished']) | set(users['in_progress'])
+
+students = []
+
+with open(path_to_file) as file_with_ids:
+    for line in islice(file_with_ids, 2, None):
+        students.append(line.split(',')[1].strip('"'))
+
+for student_id in set(students).difference(enrolled_students):
+    payload = json.dumps({'user_id': student_id})
+    requests.post(
+        'http://{HOST}:{PORT}/api/v1/course/{COURSE_ID}/user'.format(**EDXADAPT), data=payload, headers=headers
```

Comment:
http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests
> Instead of encoding the dict yourself, you can also pass it directly using the json parameter (added in version 2.4.2) and it will be encoded automatically